src/metric-engine/src/data_region.rs

```diff
@@ -142,6 +148,19 @@ impl DataRegion {
 
                 c.column_id = new_column_id_start + delta as u32;
                 c.column_schema.set_nullable();
+                match index_options {
+                    IndexOptions::Inverted => {
+                        c.column_schema.set_inverted_index(true);
```

Comment:
This is an incompatible change. Once the schema of a certain column is explicitly set to have an inverted key, the condition for indexing the tag by default will be destroyed.

Reply:
Yes... it's ok for the newly created table. But it's problematic to alter these options.

Reply:
Resolved in [2b8b907](https://github.com/GreptimeTeam/greptimedb/pull/5374/commits/2b8b9072b00950ff4616f9206d3dfa2df9228556)